app/controllers/checkins_controller.rb

```diff
@@ -128,5 +128,6 @@ def destroy
 
   def checkin_params
     params.require(:checkin).permit(:location_id, :team_id)
+    #params.require(:checkin).permit(:location_id, :team_id, :timestamp) # for when timestamp updating works
```

Comment:
If this isn't required anymore, maybe it's best to just remove the line? Or were timestamps fixed?